test/unit/stailq.c

```diff
@@ -1,13 +1,15 @@
 #include "salad/stailq.h"
 #include <stdio.h>
 #include <stdarg.h>
+
+#define UNIT_TAP_COMPATIBLE 1
```

Comment:
This was done in another commit. Please rebase.

Reply:
Done.